modules/setting/setting.go

```diff
@@ -601,6 +602,8 @@ please consider changing to GITEA_CUSTOM`)
 	ShowFooterVersion = Cfg.Section("other").Key("SHOW_FOOTER_VERSION").MustBool()
 	ShowFooterTemplateLoadTime = Cfg.Section("other").Key("SHOW_FOOTER_TEMPLATE_LOAD_TIME").MustBool()
 
+	UI.ShowUserEmailInExplore = Cfg.Section("ui").Key("SHOW_USER_EMAIL_IN_EXPLORE").MustBool()
```

Comment:
Also this line isn't necessary, see https://github.com/go-gitea/gitea/pull/336/files#diff-60f5fca6d29eafbb30eb645a0cd4041fR579